docs/_documentations/troubleshooting.md

```diff
@@ -306,7 +242,96 @@ If you have the Codewind Language Server for Node.js Profiling extension enabled
 **Workaround:** Review the preceding list and ensure that your project conforms to all of the items in the list.
 
 ***
-# Checking the application and build statuses
+# Troubleshooting project application and build statuses
+<!--
+Action/Topic: Checking the application and build statuses
+Issue type: bug
+Issue link: https://github.ibm.com/dev-ex/iterative-dev/issues/459
+18.10: Still an issue
+-->
+## Troubleshooting general application status problems
+If your application goes into the **Stopped** state unexpectedly or stays in the **Starting** state longer than expected, check the project logs to see whether something went wrong. Open the build and application logs from the project’s context menu. Problems with the build or project configuration can cause the application to fail to start. There are usually errors in the log files in this case.
+
+Even without errors, the **Stopped** state can occur if:
+  * The [health check endpoint](project-settings.html#health-check-endpoint) of the application is unreachable. Ensure it is set to an endpoint that is reachable.
+  * The [internal application port](#internal-application-port) is not correct.
+  * Your application uses HTTPS but the [HTTPS setting](project-settings.html#https-application) is not set.
+  * The time required for your application to start is longer than the [Project status ping timeout](#project-status-ping-timeout). Increase the timeout to a more sufficient time.
+
+<!--
+Action/Topic: Checking the application and build statuses
+Issue type: bug
+Issue link: https://github.ibm.com/dev-ex/iterative-dev/issues/459
+18.10: Still an issue
+-->
+## Troubleshooting general build status problems
+If your build fails, try the following:
+  * Inspect the build logs. The build logs can be opened from the project's context menu. In the build logs, you can view error messages that describe the type and cause of failure.
+  * If the build failure is related to the image build then view the docker build log and correct any errors in the project's Dockerfile.
+
+<!--
+Action/Topic: Creating a project and/or Checking the application and build statuses
+Issue type: bug/info
+Codewind version: 0.5.0
+OS: Windows
+IDE extension version: 0.5.0
+IDE version: Eclipse 2019-09
+Issue link: https://github.com/eclipse/codewind/issues/715
+-->
+## Projects stuck in starting or stopped state
+You might occasionally see projects stuck in the `Starting` or `Stopped` state even though the container logs say the projects are up and running. This can happen when you create a number of projects, for example, using the default and Appsody templates with Codewind 0.5.0. 
```

Comment:
Typically, I think it's vague to have "this" or "it" simply be the subject of the sentence. So for the second sentence, let's change it to something like:

This error can happen when you create a number of projects...
OR
This stat can occur when you create a number of projects...

Any preference between the two recommendations?

Reply:
I went with behavior because `error` and `stat` didn't feel quite right. Revision looks like:
This behavior can occur...

Can change it if you think it can be improved